tests/e2e/utils/KeycloackUrlHandler.ts

```diff
@@ -0,0 +1,32 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { TestConstants } from '../TestConstants';
+
+export class KeycloackUrlHandler {
+    public static getBaseKeycloakUrl(): string {
+        let baseKeycloakUrl: string = TestConstants.TS_SELENIUM_BASE_URL;
+
+        if (!TestConstants.TS_SELENIUM_SINGLE_HOST) {
+            baseKeycloakUrl = baseKeycloakUrl.replace('che', 'keycloak');
+        }
```

Comment:
no single-host url being returned

Reply:
I don't understand what do you mean.
If the "TS_SELENIUM_SINGLE_HOST" is false it returns multihost url
If the "TS_SELENIUM_SINGLE_HOST" is true returns single-host url

examples from logs:
 -  https://che-iokhrime-che.192.168.99.254.nip.io/auth/realms/che/protocol/openid-connect/token
 -  https://keycloak-iokhrime-che.192.168.99.254.nip.io/auth/realms/che/protocol/openid-connect/token

Reply:
I have checked it once more:
 - Single-host: https://che-iokhrime-che.192.168.99.254.nip.io/auth/realms/che/protocol/openid-connect/token
 - Multi-host  : https://keycloak-iokhrime-che.192.168.99.254.nip.io/auth/realms/che/protocol/openid-connect/token

Reply:
sorry, my bad. I misread the code.